mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -158,6 +171,21 @@ def forward(self, inputs: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
         return self.model.infer(inputs)
 
 
+class ScOpenVinoDataLoader(DataLoader):
+    def __init__(self, dataset: Dataset, inferencer: BaseOpenVINOInferencer):
+        self.dataset = dataset
+        self.inferencer = inferencer
+
+    def __getitem__(self, index):
+        image = self.dataset[index].numpy
+        annotation = self.dataset[index].annotation_scene
+        inputs, metadata = self.inferencer.pre_process(image)
+
+        return (index, annotation), inputs, metadata
+
+    def __len__(self):
+        return len(self.dataset)
+
 class OpenVINODetectionTask(IInferenceTask, IEvaluationTask):
```

Comment:
The task should implement `IOptimizationTask` interface now.

Reply:
Fixed